### ISSUE TITLE
Whether ldap v2 supports rfc4513 ?

### ISSUE THREAD

**Issue by malleyali** [go-ldap/ldap#311]
Ldap V2 supports rfc4513 or not.

**Comment by stefanmcshane** [comment]
I believe so, but @johnweldon can you confirm?
https://tools.ietf.org/html/rfc4513

**Comment by johnweldon** [comment]
I don't believe there's been any dedicated effort to ensure we support all of rfc4513, but I think most of it is supported.  We'd have to have someone actually go through the RFC and test the different mechanisms to make sure we support them all fully.
